libc/unicode/kcombiningchars.S

```diff
@@ -75,7 +75,7 @@ kCombiningCharsLz4:
 	.byte	0x00,0xa0,0x57,0x00,0x50,0xfe,0x7f,0xdf	# áW P■⌂▀
 	.byte	0xe0,0xff,0x41,0x01,0x28,0x1f,0x40,0x2f	#αλA☺(▼@/
 	.byte	0x00,0xff,0x00,0xe0,0xfd,0x66,0x00,0x00	# λ α²f  
-	.byte	0x00,0xc3,0x01,0x00,0x1e,0x00,0x64,0x20	# ├☺ ▲ d 
```

Comment:
the trailing whitespaces on this file seem alright

Reply:
I've went ahead and restored this file. Thanks for :eyes: